src/main/java/uk/gov/gchq/palisade/service/PolicyPrepopulationFactory.java

```diff
@@ -47,21 +45,20 @@
 public interface PolicyPrepopulationFactory {
```

Comment:
property ="@id" can be removed. Specifying the default value for an annotation parameter is redundant.

Reply:
Property value has been removed